input/examples/location-mitchells-hill-audiology.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Compliant with HI Vendor Test Data -->
```

Comment:
Remove the comment Compliant with HI Vendor Test Data for Location. They do not store locations. We have made some choices on how to represent a location that is associated with data that is in the vendor test data environment however the statement that this example is compliant cannot be asserted.

Reply:
Done